test/findTest.js

```diff
@@ -0,0 +1,61 @@
+var db = require('../db')
+var fs = require('fs')
+var async = require('async')
+var collectionName, result
+var done = function(err, resultArray) {result = resultArray}
+var queryFnTrue = function() {return true}
+var queryFnFalse = function() {return false}
+var expect = require('chai').expect
+
+var soldierObj1 = {
+    id: 1,
+    name: "Noa",
+    rank: "corporal",
+    limitations: "none",
+    duties: "1"
+
+}
+
+fs.mkdirSync("./collections/soldierCollection1", { recursive: true }, function(err) {
```

Comment:
chai has wonderful features called before, after, before each and aftereach. read and use accordingly 

no code should be run implicitly for the tests. all should be onside of the grnader unit test scope of this file